### PR TITLE
Added support for Olympus OMD Bulb Mode 

### PR DESCRIPTION
**Pull request by berylius**
Hi,

I made some addition to support the bulb mode for Olympus OMD cameras.
I tested it succesfully and hope  it can be merged in the main Repository.
If you have any remarks on the code don't hesitate to tell me.

Berylius

### CODE REVIEW

File: camlibs/ptp2/config.c [hunk 1]
```diff
@@ -98,7 +98,7 @@ have_prop(Camera *camera, uint16_t vendor, uint16_t prop) {
 			if ((prop & 0xf000) == 0x1000) /* generic property */
 				return 1;
 			if (camera->pl->params.deviceinfo.VendorExtensionID==vendor)
-				return 1;
+				return 1;
```
Comment (Sija) [inline camlibs/ptp2/config.c]: Ouch, please revert this line.

**Comment by msmeissn** [comment]
it would be good to avoid the whitespace changes. :/

**Comment by berylius** [comment]
Sorry 😞 ! 
Don't know how they appeared. 
There is still one change I don't understand an it's library.c line 1, I don't see any diff!?

**Comment by msmeissn** [comment]
i will go and merge it ... 

I will also remove or repalce the // Modif comments from you if this is ok

**Comment by berylius** [comment]
Yes no problem.

File: camlibs/ptp2/config.c [hunk 2]
```diff
@@ -2460,6 +2460,89 @@ _put_Olympus_ISO(CONFIG_PUT_ARGS)
 	}
 	return GP_ERROR;
 }
+//Modif Berylius: Add OMD Drive Mode Widget
+/* Work in progress
```
Comment (Sija) [inline camlibs/ptp2/config.c]: What about this part?

Comment (berylius) [reply camlibs/ptp2/config.c]: I started it but i need to sniff the USB trafic to finish the function. 
It has no impact on the Bulb mode function (main reason for the pull request)

Comment (Sija) [reply camlibs/ptp2/config.c]: Therefore I'd remove it for the time being and re-introduce it in another PR once you'll get things working.

**Comment by msmeissn** [comment]
I merged it manually, including a bit of whitespace cleanup and some small fixes.

**Comment by msmeissn** [comment]
merged manually :)

**Pull request closed without merging**